app/src/main/java/com/andela/app/animationchallenge/view/SignUpActivity.java

```diff
@@ -165,6 +165,11 @@ public void onComplete(@NonNull Task<AuthResult> task) {
                             Log.d(TAG, "createUserWithEmail:success");
                             showMessage("Account created");
                             FirebaseUser user = mAuth.getCurrentUser();
+                            if (user != null) {
```

Comment:
The goal wasn't to save directly the user object provided by Firebase Auth, You have to use the User class that we create manually in the model package